src/mesh_sandbox/handlers/inbox.py

```diff
@@ -82,6 +82,7 @@ def _get_response_headers(message: Message, chunk_number: int):
             Headers.Mex_To: message.recipient.mailbox_id,
             Headers.Mex_WorkflowID: message.workflow_id,
             Headers.Mex_Chunk_Range: f"{chunk_number}:{message.total_chunks}",
+            Headers.Mex_Total_Chunks: f"{message.total_chunks}",
```

Comment:
why is this a string  not an integer ?  and can this be None?

Reply:
has to be a string and total_chunks shouldn't ever be none. i've updated it with and "or 0" just in case though, as you advised. in this commit: a4ffbe66142364dbc8b5306b420c8cbf7ca413e1